src/mqtt.h

```diff
@@ -0,0 +1,25 @@
+#ifndef MQTT_H
+#define MQTT_H
+
+#include "string.h"
+#include "MQTTClient.h"
+
+
+typedef struct{
+        // MQTT init
+    MQTTClient client;
+    MQTTClient_connectOptions conn_opts ;
+    MQTTClient_message pubmsg;
```

Comment:
brauchst du das hier drinnen musst du dass nicht sowieso in der loop jedesmal neu erstellen